solutions/cses-1132.mdx

```diff
@@ -80,7 +83,98 @@ int main() {
 	}
 	return 0;
 }
+```				 
+</CPPSection>
+	
+<JavaSection>
+	
+<Warning>
+Java exceeds the time limit on two test cases.
+</Warning>
+	
+```java
+import java.util.*;
+import java.io.*;
+
+public class TreeDistancesI {
+	// source -> https://cses.fi/problemset/task/1132/
+
+	static ArrayList<Integer>[] adj;
+	static int [] d1 = new int[200001];  // to store first-max length.
+	static int [] d2 = new int[200001]; // to store second-max length.
+	static int [] c = new int[200001]; // to store child for path of max length.
```

Comment:
Can you make more descriptive variable names so it's easier to understand the rest of the code?

Reply:
make comment spacing consistant